src/sentry/models/sentryappwebhookerror.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+
+from django.db import models
+from django.utils import timezone
+
+from sentry.db.models import EncryptedJsonField, FlexibleForeignKey, Model
+
+
+class SentryAppWebhookError(Model):
+    __core__ = False
+
+    date_added = models.DateTimeField(default=timezone.now)
+
+    sentry_app = FlexibleForeignKey("sentry.SentryApp", related_name="webhook_errors")
+
+    organization = FlexibleForeignKey(
+        "sentry.Organization", related_name="sentry_app_webhook_errors"
+    )
+
+    # If there is a related Sentry error, store it here
+    error_id = models.CharField(max_length=64, null=True)
```

Comment:
I'm not sure what it should be. We haven't figured out if it's even possible to link this to a Sentry error yet so this field is unclear for now

Reply:
Should it just be removed?

Reply:
@nolachen I think it might make more sense to make a join table that links the `error_id` to the `SentryAppWebhookError` rather than putting in the model of `SentryAppWebhookError`.